plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -170,6 +170,77 @@ func (n *NodeLabelPrioritizer) CalculateNodeLabelPriority(pod *api.Pod, machines
 	return result, nil
 }
 
+// This is a reasonable size range of all container images. 90%ile of images on dockerhub drops into this range.
+const minImgSize int64 = 23000000   // min 23MB
```

Comment:
Can you please declare a constant mb = 1024 * 1024 (or 1000 * 1000 works as well) and define this and the next constant as 23 * mb and 1000 * mb. It is more readable that way and easy to catch typos. Example: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/disk_manager.go#L32